dbflute-engine/src/main/java/org/dbflute/logic/doc/spolicy/DfSPolicyColumnStatementChecker.java

```diff
@@ -205,6 +209,14 @@ protected void evaluateColumnThenTheme(DfSPolicyStatement statement, DfSPolicyRe
             if (!column.hasClassification() == !notThenClause) {
                 result.violate(policy, "The column should " + notOr + "be classification: " + toColumnDisp(column));
             }
+        } else if (clsMatcher.find()) {
```

Comment:
The clsMatcher instance might be unused.
What about calling matcher() after thenTheme.startsWith("classification")?

Reply:
> What about calling matcher() after thenTheme.startsWith("classification")?
It needs to ignore case so like this:
```
Srl.startsWithIgnoreCase(thenTheme, "classification")
```